control/rlocus.py

```diff
@@ -46,6 +46,10 @@ def root_locus_map(sysdata, gains=None):
     gains : array_like, optional
         Gains to use in computing plot of closed-loop poles.  If not given,
         gains are chosen to include the main features of the root locus map.
+    xlim : array_like, 2 elements, optional
+        Plot range
```

Comment:
Add period. Perhaps also expand description (see other uses of xlim, for example in `control.root_locus_plot`.